ext/session/mod_files.c

```diff
@@ -581,7 +581,7 @@ PS_UPDATE_TIMESTAMP_FUNC(files)
 	}
 
 	/* Update mtime */
-#ifdef HAVE_UTIME_NULL
+#ifdef HAVE_UTIME
```

Comment:
Is this necessary? We're unconditionally using VCWD_UTIME below.

Reply:
Interestingly, `VCWD_UTIME` is defined only if we `HAVE_UTIME`!